### PR TITLE
use the url template tag instead of a context variable

### PR DESCRIPTION
**Pull request by benjaoming**
A fairly simple change :) And maybe more a small example or madman OCD. But putting a link somewhere is template logic, not view logic :)

### CODE REVIEW

File: kalite/facility/templates/facility/login.html [hunk 1]
```diff
@@ -25,7 +25,8 @@ <h3 class="no-margin-top">{% trans "Log in" %}</h3>
         </form>
         <div id="info_block">
             {% if not restricted %}
-                {% blocktrans %}If you don't have a login, please <a href="{{ sign_up_url }}">sign up</a>.{% endblocktrans %}
+                {% url 'facility_user_signup' as signup_url %}
+                {% blocktrans %}If you don't have a login, please <a href="{{ signup_url }}">sign up</a>.{% endblocktrans %}
```
Comment (jamalex) [inline kalite/facility/templates/facility/login.html]: Where is the `signup_url` variable defined?

Comment (benjaoming) [inline kalite/facility/templates/facility/login.html]: `{% url 'facility_user_signup' as signup_url %}` ;)

Comment (jamalex) [inline kalite/facility/templates/facility/login.html]: Oh, totally didn't even see that line. Any reason the url blocktag can't just go right inside the href? I guess this is cleaner for i18n?

Comment (benjaoming) [inline kalite/facility/templates/facility/login.html]: Yeah, it's a "feature" of blocktrans that you cannot put other block tags inside it.. just simple variable substitution. Not even template filters are allowed.

**Comment by aronasorman** [comment]
Can you put this into the https://github.com/learningequality/ka-lite/wiki/Coding-guidelines-and-conventions wiki article @benjaoming?

**Comment by benjaoming** [comment]
Absolutely :) I have a small notebook on "things that could improve over time" :) So I'm hoping we can have time for a Django oriented session together next week to talk about best practices, conventions, coding patterns etc...

**Comment by benjaoming** [comment]
Rearranged the article a bit by putting a Django chapter at the bottom and including the existing template paragraph in that one.

I wrote the MVT chapter... let's talk more about these things soon!

**Pull request merged**